test/quadquizaminos/contests_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule Quizquadaminos.ContestsTest do
+  use Quadquizaminos.DataCase
+  alias Quadquizaminos.Contests
+
+  setup do
+
+    {:ok, contest} = Contests.create_contest(%{name: "ContestC"})
+    {:ok, {:ok, started_contest}}= Contests.start_contest("ContestC")
```

Comment:
Why do we need to start the contest on the setup? Considering that setup is used for all the preceding tests which means that for all the tests we create both cotests and start one of them.

I dont mean this is a wrong approach, I am curious to know why 😄

Reply:
Because in every test we need a contest that was created and had already been started